src/lib/feature-flags/service.ts

```
@@ -0,0 +1,420 @@
+      };
+      this.logUsage(flagKey, result);
+      return result;
+    }
+
+    // Handle different flag types
+    let enabled = false;
+    let reason = '';
+
+    switch (flag.type) {
+      case 'boolean':
+        const boolFlag = flag as BooleanFeatureFlag;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)